composer.json

```diff
@@ -23,7 +23,7 @@
     "require": {
         "php": "^7.1",
         "cocur/slugify": "^1.0 || ^2.0 || ^3.0",
-        "doctrine/doctrine-bundle": "^1.0",
+        "doctrine/doctrine-bundle": "^2.0",
```

Comment:
I think you will have to use `^1.0 || ^2.0` to have the build pass on `--prefer-lowest` and on the sf `3.4` (those are the Travis jobs that are currently failing)